src/main/java/org/broadinstitute/hellbender/utils/GenotypeUtils.java

```diff
@@ -62,9 +62,19 @@ public static GenotypeCounts computeDiploidGenotypeCounts(final VariantContext v
                 continue;
             }
 
-            if (!g.hasLikelihoods() && g.isHomRef()) {
+            if (!g.hasLikelihoods()) {
```

Comment:
I find the control flow kind of confusing here now.  I might rearrange it into a tree with round vs not round branches instead of pulling out that error case at the top.

Something like this:

```
            if (!g.hasLikelihoods()) {
                if (roundContributionFromEachGenotype) {
                    if (g.isHomRef()) {
                        genotypeWithTwoRefsCount += 1;
                    } else if (g.isHet()) {
                        genotypesWithOneRefCount += 1;
                    } else {
                        genotypesWithNoRefsCount += 1;
                    }
                } else {
                    if (!g.isHomRef()) {
                        throw new IllegalStateException("Genotype likelihoods cannot be determined from GQ for genotype: " + g +
                                ".\nGenotypes with no PLs should have integer counts using roundContributionFromEachGenotype = true.");
                    }
                    if (gq == 0) {
                        genotypeWithTwoRefsCount += 1.0/3;
                        genotypesWithOneRefCount += 1.0/3;
                        genotypesWithNoRefsCount += 1.0/3;
                    } else {
                        genotypeWithTwoRefsCount += QualityUtils.qualToProb(gq);
                        genotypesWithOneRefCount += 1 - QualityUtils.qualToProb(gq);
                        //assume last likelihood is negligible
                    }
                }
                continue;
            }
```